frontend/yarn.lock

```diff
@@ -11209,8 +11209,8 @@ typedarray@^0.0.6:
   resolved "https://registry.npmjs.org/typedarray/-/typedarray-0.0.6.tgz"
   integrity "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c= sha512-/aCDEGatGvZ2BIk+HmLf4ifCJFwvKFNb9/JeZPMulfgFracn9QFcAf5GO8B/mweUjSoblS5In0cWhqpfs/5PQA=="
 
-typescript@^4.9.5:
-  version "4.9.5"
+typescript@^4.1.2:
+  version "4.1.2"
```

Comment:
Don't think was intentional ; could we revert this?

Reply:
we use version 4.1.2 for ts, I upgraded it to 4.9.5 in an earlier PR but never updated the yarn.lock file and caught it here

Reply:
Ahhh gotcha ok, resolving in that case